packages/teams-js/src/public/interfaces.ts

```diff
@@ -221,9 +236,13 @@ export interface SecondaryId {
  * @beta
  */
 export enum SecondaryM365ContentIdName {
+  /** One drive ID */
   DriveId = 'driveId',
+  /** Teams Group ID */
   GroupId = 'groupId',
+  /** SharePoint ID */
   SiteId = 'siteId',
+  /** User ID */
```

Comment:
>User ID [](http://example.com/codeflow?start=6&length=7)

What format is this user ID in?

Reply:
ADAIK user id can be a string, I can add it here.

Reply:
But what kind of user ID? Is this an AAD ID?

Reply:
After hunting down some docs and Jess's help User ID is a GUID. Will update in the latest commit.